### PR TITLE
Fixed how ControlMeasureLogData is handled in ControlNetVersioner

### PR DESCRIPTION
**Pull request by jessemapel**
We still need to remove the old constructor and ToProtobuf methods

### CODE REVIEW

**Comment by jessemapel** [comment]
Don't review this yet! I just realized I forgot to handle the v1 proto case

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 1]
```diff
@@ -1756,24 +1759,21 @@ namespace Isis {
...
 
-          logData.set_doubledatatype(log.GetDoubleDataType());
-          logData.set_doubledatavalue(log.GetDoubleDataValue());
-          logData.set_booldatatype(log.GetBoolDataType());
-          logData.set_booldatavalue(log.getBoolDataValue());
+          logData.set_doubledatatype( (int) log.GetDataType() );
+          logData.set_doubledatavalue( log.GetNumericalValue() );
 
-          protoMeasure.add_log(logData);
+          *protoMeasure.add_log() = logData;
```
Comment (AgoinsUSGS) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Is that syntax right?

Comment (jessemapel) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Yeah. It's a bit of a fudge. add_log returns a pointer to the new protobuf log data object. This then dereferences it and assigns to it.

Comment (jessemapel) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: looking through control net versioner, we're gonna have to tweak how we set the covariance matrices and the measures

Comment (jessemapel) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: For POD types like double, protobuf allows for calling add_fieldname(value). So we'll need to change how we add measures, but not stuff like the covariance matrix.

Comment (AgoinsUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Interesting. Looks good

**Pull request merged**